src/lang/modifyAst.test.ts

```diff
@@ -890,11 +890,15 @@ sketch002 = startSketchOn({
         codeBefore.indexOf(lineOfInterest),
         codeBefore.indexOf(lineOfInterest) + lineOfInterest.length,
       ]
+      const artifact = { type } as any
```

Comment:
This is still unsafe, but at least it will be caught if the function changes. Remember to import `Artifact`.

```suggestion
      const artifact = { type } as Artifact
```